src/main/java/core/basesyntax/service/db/Options.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.db;
+
+import core.basesyntax.service.operation.BalanceFruitService;
+import core.basesyntax.service.operation.FruitService;
+import core.basesyntax.service.operation.PurchaseFruitService;
+import core.basesyntax.service.operation.ReturnFruitService;
+import core.basesyntax.service.operation.SupplyFruitService;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Options {
```

Comment:
think what to do with this database is not the best idea. Let's move this topic to the strategy